src/token.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * A Token class
+ * @constructor
+ * @param {string} owner - The player who owns this token.
+ * @param {string} [color=COLOR_BLACK] - Token color. See constants.js.
+*/
+function Token(owner, space, color) {
+  this.owner = owner;
+  this.space = space;
+  this.color = color;
+  this.isDead = false;
```

Comment:
board.tokens keeps a reference to all tokens, and view.tokenViews has a reference to each one as well. killing it removes it from the users tokens but not the other 2... view detects that it's killed and removes it from it's view... board never removes it 

^ that's how i did it, not sure if it's the best way. maybe it'd be cleaner to just remove all the references and tell the view to destroy that view